_posts/2019-12-01-rle-array.markdown

```diff
@@ -212,7 +212,7 @@ Often we find columns in our DataFrames where information only occurs for a very
 is often more efficient to explicitly store and look-up these few cases --- e.g. by using a
 [HashTable](https://en.wikipedia.org/wiki/Hash_table) --- than using a simple array:
 
-![sparse data memory layout](/assets/images/2019-12-01-rle-array/sparse_data.png)
+![sparse data memory layout](/assets/images/2019-12-01-rle-array/sparse_data.svg)
```

Comment:
Just a nitpick and the only thing that catched my eye: the space between the "values" and "mapping" blocks looks a bit too big, they seem "unrelated". But again, just a nitpick.